visualrecognitionv4/visual_recognition_v4.go

```diff
@@ -2429,7 +2429,6 @@ type UpdateObjectMetadata struct {
 func (visualRecognition *VisualRecognitionV4) NewUpdateObjectMetadata(object string, count int64) (model *UpdateObjectMetadata, err error) {
 	model = &UpdateObjectMetadata{
 		Object: core.StringPtr(object),
-		Count:  core.Int64Ptr(count),
```

Comment:
Any reason count is removed?

Reply:
In the current generations, the `count` property is removed, because it is a property of the response and not the request. However, to avoid any unforeseen issues, I am reverting this change. We can address it in an upcoming major release.